src/components/Structure/Summary/index.js

```diff
@@ -130,24 +152,25 @@ export class SummaryStructure extends PureComponent /*:: <Props> */ {
             <div className={f('medium-3', 'columns')}>
               <div className={f('panel')}>
                 <h5>External Links</h5>
+                <p>
+                  View <code>{metadata.accession}</code> in:
+                </p>
```

Comment:
I don't think this is necessary, as the PDB accession is already displayed in three other places on the page.